app/Providers/RouteServiceProvider.php

```diff
@@ -69,13 +70,30 @@ private function mapAppRoutes()
 
     private function mapAuthRoutes(): void
     {
+        Route::middleware(['app'])
+            ->namespace($this->namespace)
+            ->prefix('auth')
+            ->group(
+                function () {
+                    //TODO: move out from 'users' - it is being used in API
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)